MyPerfectOnboardingApplication/Sources/MyPerfectOnboarding.Contracts/Services/ListItems/IAdditionService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using MyPerfectOnboarding.Contracts.Models;
+
+namespace MyPerfectOnboarding.Contracts.Services.ListItems
+{
+    public interface IAdditionService
+    {
+        Task<Models.ListItem> AddItemAsync(Models.ListItem item);
```

Comment:
```suggestion
        Task<Models.ListItem> AddItemAsync(ListItem item);
```
(otherwise, you have an unused using (ref https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/4#discussion_r260338413 & https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/4#discussion_r260339457) here 😉)